api/app/routes/chat.py

```
@@ -79,47 +82,46 @@
             logger.error(f"Validation error details: {str(e)}")
             # Use model_json_schema instead of model_fields
             logger.error(
-                f"Model fields: {QueryRequest.model_json_schema()['properties']}")
+                f"Model fields: {QueryRequest.model_json_schema()['properties']}"
+            )
             raise HTTPException(status_code=422, detail=str(e))
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)